akd/src/append_only_zks.rs

```diff
@@ -790,6 +787,66 @@ mod tests {
         Ok(())
     }
 
+    #[tokio::test]
+    #[allow(dead_code)]
+    async fn test_nonmembership_proof_v_small() -> Result<(), AkdError> {
+        let num_nodes = 2;
+
+        let mut insertion_set: Vec<Node<Blake3>> = vec![];
+
+        for i in 0..num_nodes {
+            let mut label_arr = [0u8; 32];
+            label_arr[31] = u8::from(i);
+            let label = NodeLabel::new(label_arr, 256u32);
+            let mut input = [0u8; 32];
+            input[31] = u8::from(i);
+            let hash = Blake3Digest::new(input);
+            let node = Node::<Blake3> { label, hash };
+            insertion_set.push(node);
+        }
```

Comment:
Could we move this part to a separate helper function?

Reply:
What about `generate_random_node()`? 

If we want a specific property for the node, let's add this as a comment.